### PR TITLE
Issue 10 : Made the footer responsive for all devices

### LINKED ISSUE #10: SeatAndEat : Make Footer Responsive
- Make the footer responsive for all devices

### PR DESCRIPTION
**Pull request by iamatifmoin**
## Linked Issue 

Closes: #10

### Describe the changes you've made

1. Made the footer responsive for all devices
2. Enhanced the styling of the footer

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 **On Smaller Displays:**

footer-small

**On Larger Displays:**

footer-big

### CODE REVIEW

File: styles/footer.css [hunk 1]
```diff
@@ -12,27 +12,27 @@ footer {
...
+    padding-left: 1.5em;
+}
+
 .footer-container {
+    padding-top: 1.5em;
+    padding-bottom: 1.5em;
     display: grid;
     grid-template-columns: repeat(3, 1fr);
-    gap: 4rem;
+    gap: 1.5rem;
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Don't change the grip gap. let it be 4rem.

File: styles/footer.css [hunk 2]
```diff
@@ -12,27 +12,27 @@ footer {
...
     /* justify-content: center; */
     /* align-items: center; */
-    padding: 2rem 4rem;
     width: 100%;
     max-width: 1200px;
-    margin: auto;
+    margin: 0;
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Don't change the Margin from auto to 0.

File: styles/footer.css [hunk 3]
```diff
@@ -12,27 +12,27 @@ footer {
     border-top: 1px solid white;
 }
 
+.contact-us{
+    padding-left: 1.5em;
+}
+
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Remove this padding from contact-us container.

File: styles/footer.css [hunk 4]
```diff
@@ -12,27 +12,27 @@ footer {
...
+    gap: 1.5rem;
     /* justify-content: center; */
     /* align-items: center; */
-    padding: 2rem 4rem;
     width: 100%;
     max-width: 1200px;
-    margin: auto;
+    margin: 0;
     color: var(--clr-light);
-    line-height: 1.5;
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Don't remove the line height.

File: styles/footer.css [hunk 5]
```diff
@@ -12,27 +12,27 @@ footer {
...
 }
 
-.footer-container a {
-    color: var(--clr-light);
-}
 
 .footer-container h2 {
     font-size: 1.75rem;
-    letter-spacing: 2px;
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Don't remove the letter spacing.

File: styles/footer.css [hunk 6]
```diff
@@ -12,27 +12,27 @@ footer {
...
     width: 100%;
     max-width: 1200px;
-    margin: auto;
+    margin: 0;
     color: var(--clr-light);
-    line-height: 1.5;
 }
 
-.footer-container a {
-    color: var(--clr-light);
```
Comment (Rupam-Matabber) [inline styles/footer.css]: Don't change this color.

**Review by Rupam-Matabber — COMMENTED** [review]
Please change the highlighted sections. In this issue there is no need to change any existing design but why you have changed them?

**Comment by iamatifmoin** [comment]
@Rupam-Matabber I have made the changes you suggested.

**Pull request merged**